l1-contracts/test/foundry/unit/concrete/Bridgehub/experimental_bridge.t.sol

```diff
@@ -71,8 +73,45 @@ contract ExperimentalBridgeTest is Test {
         assertEq(bridgeHub.owner(), bridgeOwner);
     }
 
+    function test_newPendingAdminReplacesPrevious(address randomDeployer, address otherRandomDeployer) public {
+        assertEq(address(0), bridgeHub.admin());
+
+        if (randomDeployer == otherRandomDeployer) {
+            return;
+        }
```

Comment:
I think you should use `assume`:
- https://book.getfoundry.sh/cheatcodes/assume

Reply:
fixed